item.py

```diff
@@ -0,0 +1,21 @@
+class Item():
```

Comment:
This is just semantics but maybe we should call these "towers"? It's fine for now but I wanna get your temperature on changing it in future

Reply:
yup, item was just a placeholder